tests/e2e/scenarios/lib/common.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+echo "CLOUD_PROVIDER=${CLOUD_PROVIDER}"
+echo "CLUSTER_NAME=${CLUSTER_NAME}"
+
+if [[ -n "${KOPS_BASE_URL-}" ]]; then
+    echo "KOPS_BASE_URL was unexpectedly set. Please unset."
+    exit 1
+fi
+
+export KOPS_FEATURE_FLAGS="SpecOverrideFlag,${KOPS_FEATURE_FLAGS:-}"
+export GO111MODULE=on
+
+if [[ -z "${AWS_SSH_PRIVATE_KEY_FILE-}" ]]; then
+    export AWS_SSH_PRIVATE_KEY_FILE="~/.ssh/id_rsa"
+fi
+if [[ -z "${AWS_SSH_PUBLIC_KEY_FILE-}" ]]; then
+    export AWS_SSH_PUBLIC_KEY_FILE="~/.ssh/id_rsa.pub"
+fi
```

Comment:
not a blocker:

kubetest2-kops recognizing those flags was intended for backwards compatibility with the prow job env vars

https://github.com/kubernetes/test-infra/blob/2fd611583393415eda044de79eb27d7fd816386c/config/prow/config.yaml#L666-L683

I wish they didn't have AWS_ in the name given that they're provider-agnostic. Maybe we could specify the flags instead? or have kubetest2-kops also look for a more generic set of env var names? 

```
      --ssh-private-key string           The path to the private key used for SSH access to instances
      --ssh-public-key string            The path to the public key passed to the cloud provider
```

Reply:
When I test these scripts locally I wanted to emulate the prow environment. I was a bit torn between doing the above or demanding those vars being set (because I always forget to export all that kubetest2 expects).

Reply:
yea that makes sense to me. we can revisit this later